src/logic/otp2/controller.ts

```diff
@@ -576,20 +579,43 @@ export async function searchTransit(
 
     // initial search date may differ from search date if this is a
     // continuation search using a cursor.
-    const isFirstSearchIteration = initialSearchDate === searchDate
+    const isFirstSearchIteration = isEqual(
+        setMilliseconds(initialSearchDate, 0),
```

Comment:
Ser for meg at dette kan feile dersom millisekund-differansen gjer at den eine datoen får eit anna sekund enn den andre datoen. Er det bedre å sjekke på differanse?
```
const isFirstSearchIteration = differenceInSeconds(initialSearchDate, searchDate) < 1
```

Reply:
Ja, fint. Kan gjerne være minutt også.